scripts/generate-metadata

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+# Original Source from: https://github.com/GrapheneOS/scripts
```

Comment:
Just make this commit as @strcat with --author. Copy-pasting work of others without at least git-commit credit is a pet peeve of mine. The rest of this looks fine though.

Reply:
Done